src/app/config.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ConfigService {
+
+  constructor() { }
+
+  fileName = 'assets/eurofxref-daily.xml';
```

Comment:
I think `fileName` should be declared `read-only`.
It would be better to define these type of variables in the environment settings.